draft-rswg-rfc7990-updates.mkd

```diff
@@ -96,8 +96,24 @@ and by dropping the requirement that the XML be embedded in the PDF publication
 
 Historical text from {{RFC7990}} such as Section 2 ("Problem Statement"), Section 4 ("Overview of the Decision-Making Process"), and Section 10 ("Transition Plan") are not copied to this document.
 
-Section 7.6 of "RFC Editor Model (Version 3)" {{RFC9280}} says "Once published, RFC Series documents are not changed."
-{{updating}} and {{pub-versions}} in this document update that policy.
+## Changes to RFC 9280 {#changes-to-9280}
+
+Section 7.6 of {{RFC9280}} currently says:
+
+> Once published, RFC Series documents are not changed.
+
+This document replaces that sentence with:
+
+> Once published, RFC Series documents may be re-issued, but unintended changes shall be avoided, and content semantics are to be preserved.
```

Comment:
Do you really think that "unintended changes shall be avoided" is worth saying?

```suggestion
> Once published, RFC Series documents may be re-issued, but the semantic content of published documents shall be preserved.
```

Reply:
I agree with Martin here. Unintended everything is always avoided.

Reply:
That's fair.